src/components/AddBookForm.js

```diff
@@ -1,13 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { addBook } from '../redux/books/books';
 import './AddBookForm.css';
 
 export default function AddBookForm({ categories }) {
+  const [bookTitle, setBookTitle] = useState('');
+
```

Comment:
Please ensure that you have a local state with  `title` and `author` according to the project requirement.

Reply:
@micheaol Thanks for your review. But there is nothing about author. We have no author field in the design and there is no clear instruction from where we should get it. So I have hard coded author name.

Reply:
@ahangarha,

You can set the local state to a value or empty string, then update the state onChange. I hope this helps.

Reply:
Update it based on what? We don't have any input in the design for author name

Reply:
@micheaol This is the design
![image](https://user-images.githubusercontent.com/11241315/160483711-9658d804-42bd-4628-a7b9-1e67e3a7c0a1.png)

Do you suggest I add another field for author?

Reply:
@ahangarha,

I understand that there is no input for the author in the design. But we can set the value in the local state to any value we want, then pass it when you update (onChange). I hope this helps.

Reply:
@micheaol 
Ok. I did that and I think I should request for a new CR. Right?